Gruntfile.js

```diff
@@ -1,35 +0,0 @@
-var cheerio = require('cheerio')
-
-module.exports = function (grunt) {
-  grunt.loadNpmTasks('grunt-link-checker');
-
-  grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
-
-    linkChecker: {
```

Comment:
Okay, I brought `Gruntfile.js` back, along with its deps. Can you describe how you use it, so I can add a note to the README?